less/pageLevelProgress.less

```diff
@@ -8,6 +8,8 @@
         &.drawer-item-open.disabled {
             background-color:@item-color-disabled;
             color:@item-text-color-disabled;
+            font-size: @button-text-font-size;
```

Comment:
we can not use this variable in the plugin
it is defined only in the vanilla theme version 2, so build will break with older versions of the theme